### PR TITLE
login and register form styling 

### PR DESCRIPTION
**Pull request by katlazarova**

### CODE REVIEW

File: web/themes/custom/pineapple/components/_button.twig [hunk 1]
```diff
@@ -1 +1 @@
-<a href="{{ button_url }}" class="btn btn-link">{{ button_text }}</a>
+<a href="{{ button_url }}" class="btn button-link">{{ button_text }}</a>
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/components/_button.twig]: should btn not be button?

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    margin: $unit;
+    display: block;
+    width: 60%;
+
+    @include breakpoint($lap) {
+      display: inline;
+      width: 30%;
+    }
+  }
+}
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/forms/_form-items.scss]: Remove this entire section

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.user-register-form,
+.user-login-form {
+  background-color: $white;
+  color: $dark-gray;
+  font-weight: bold;
+  font-family: "Lato", sans-serif;
+  text-transform: capitalize;
+  padding: $unit * 2;
+  font-size: 12px;
+  width: 70%;
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/forms/_form-items.scss]: These forms should be 100% width on mobile

File: web/themes/custom/pineapple/components/_button.twig (hunk 1 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/components/_button.twig]: yes

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss (hunk 2 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/forms/_form-items.scss]: done

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss (hunk 3 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/forms/_form-items.scss]: done

File: web/themes/custom/pineapple/sass/base/_buttons.scss [hunk 4]
```diff
@@ -1,51 +1,53 @@
...
   vertical-align: middle;
   border-radius: 3px;
   transition: 0.5s;
 }
 
-.btn-primary {
-  background-color: $light-green;
+.button--primary {
+  background-color: $gray;
+  color: white;
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/_buttons.scss]: This should be a variable

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss [hunk 5]
```diff
@@ -0,0 +1,55 @@
+.user-register-form,
+.user-login-form {
```
Comment (ElXyfer) [inline web/themes/custom/pineapple/sass/base/forms/_form-items.scss]: There is gonna be other forms in the near future do you also want them to have this styling?

File: web/themes/custom/pineapple/sass/base/_buttons.scss (hunk 4 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/_buttons.scss]: done

File: web/themes/custom/pineapple/sass/base/forms/_form-items.scss (hunk 5 above)
Comment (katlazarova) [reply web/themes/custom/pineapple/sass/base/forms/_form-items.scss]: understood

**Pull request merged**